### ISSUE TITLE
Error while installing using pip / pip3

### ISSUE THREAD

**Issue by varadgunjal** [luispedro/mahotas#80]
1) Ubuntu 16.04, g++ (Ubuntu 5.4.0-6ubuntu1~16.04.2) 5.4.0

Tried installing with `pip install mahotas` & `sudo pip install mahotas`. And corresponding commands for pip3 and tried the same on the package from PyPI as well. Get the following error message - 

```
Collecting mahotas
  Downloading mahotas-1.4.2.tar.gz (1.5MB)
    100% |████████████████████████████████| 1.5MB 456kB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-65OM5L/mahotas/setup.py", line 50, in <module>
        long_description = open('README.md').read()
    IOError: [Errno 2] No such file or directory: 'README.md'
    
    ----------------------------------------
```

**Comment by luispedro** [comment]
This was a packaging mistake. I just fixed it and release version 1.4.3.

Sorry

**Comment by varadgunjal** [comment]
Thanks!
